packages/lesswrong/integrationTests/schemacheck.test.ts

```diff
@@ -0,0 +1,9 @@
+import "./integrationTestSetup";
+import path from "path";
+import { makeMigrations } from "../server/scripts/makeMigrations";
+
+describe('Schema check', () => {
+  it('Has an accepted_schema.sql file which matches the schema defined in code', async () => {
+    await makeMigrations({writeSchemaChangelog: false, writeAcceptedSchema: false, generateMigrations: false, rootPath: path.join(__dirname, "../../../")});
```

Comment:
It might make sense to split this out into it's own CI stage at some point so it runs faster (it will be annoying to wait for all the tests to pass only to find that you forgot to run `yarn acceptschema`), but let's wait and see how much of a problem this is

Reply:
Good shout! This is another thing I tried and failed to get to work before but I have now managed it by doing `require('../server.ts');`